addons/pos_loyalty/static/src/overrides/components/order_summary/order_summary.xml

```diff
@@ -32,7 +32,6 @@
                         </div>
                     </t>
                 </t>
-            </div>
```

Comment:
you can now indent the whole block correctly

Reply:
@vlst-odoo updated the indentation

Reply:
hi @vlst-odoo ,
is it good to get merged now ?